### ISSUE TITLE
Spellslinger support

### ISSUE THREAD

**Issue by Razunter** [xanthics/PoE_Weighted_Search#24]
When you have skill supported by [spellslinger](https://pathofexile.gamepedia.com/Spellslinger), site should include attack mods on a wand when selected "caster weapon". Currently selecting "attack" type doesn't help.

**Comment by xanthics** [comment]
That's a tricky one because there is no way for weights to account for the base weapon, which can account for a difference of about 45 flat phys.

Although something is certainly better than nothing, so I'll see what I can work out with @coldino

**Comment by xanthics** [comment]
> Supported Skills have Added Spell Damage equal to (50–100)% of Damage of Equipped Wand
> If two Wands are Equipped, each contributes half as much Added Damage

Adding this here as a reminder of the mechanic. As at first glance it seems difficult to model exact numbers, although relative numbers should be possible.

**Comment by Razunter** [comment]
Well, at least add "Pseudo % [element] to attacks" to search query when there is already "Pseudo % [element] to spells". This will work for most cases.

**Comment by xanthics** [comment]
In order to accurately model a spellslinger weapon, the following mods need to be considered(in addition to all caster mods)

Flat phys (local)
% increased phys (local, multiplier to previous plus base))
% quality (local, additive with previous)
Flat Elemental/chaos (local, all 4 elements)

Which essentially means the phys part can be off by 45 flat * %ipd.  And the rest of it can be off based on quality (corrupted 0% vs beast/fossil 30%)

--------------

Basically the elemental part is easy, but the physical part is hard.  Further complicated if the user want to actually consider their wand attacks as well, although then they are just looking for the highest dps weapon.

I have work to finish today, but I'll circle back around to this tonight.  I've also pinged Novynn about it.
